@vates/types/src/xo.mts

```diff
@@ -211,8 +212,21 @@ export type XoHostPatch = BaseXapiXo & {
 }
 
 export type XoNetwork = BaseXapiXo & {
+  automatic: boolean
+  bridge: string
+  current_operations?: Record<string, NETWORK_OPERATIONS>
+  defaultIsLocked: boolean
   id: Branded<'network'>
+  insecureNbd?: boolean
+  MTU: number
+  name_description: string
+  name_label: string
+  nbd?: boolean
+  other_config: Record<string, string>
+  PIFs: XoPif['id'][]
+  tags: string[]
   type: 'network'
+  VIFs: XoVif['id'][]
```

Comment:
Try to keep properties sorted.

- Special char ($, _,...)
- Uppercase prop (PIFs, VIFs,...)
- other properties

You can check `BaseXoVm` if you need some example